config/routes.rb

```diff
@@ -23,4 +25,5 @@
     resources :items, only: [:index, :show, :edit, :new, :create, :update]
   end
 
+  get '*unmatched_route', to: 'application#not_found'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.